Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -35,6 +35,7 @@ open class MessageContentCell: MessageCollectionViewCell {
         let containerView = MessageContainerView()
         containerView.clipsToBounds = true
         containerView.layer.masksToBounds = true
+        containerView.isUserInteractionEnabled = true
```

Comment:
`UIImageView` `isUserInteractionEnabled` is off by default, but in this case it is a container view.